custom/icds_reports/tasks.py

```diff
@@ -834,3 +842,14 @@ def push_missing_docs_to_es():
             from_email=settings.DEFAULT_FROM_EMAIL,
             recipient_list=["{}@{}.com".format("jmoney", "dimagi")]
         )
+
+@periodic_task(run_every=crontab(hour=23, minute=0, day_of_month='15'), acks_late=True, queue='icds_aggregation_queue')
```

Comment:
I think that this should be the 14th. Also this will use the aggregate data from the 13th because the tables aggregation gets it from agg_ccs_record, not sure if that was intended